docs/code-search/code-navigation/inference_configuration.mdx

```diff
@@ -103,7 +103,8 @@ This auto-indexing-specific library defines the following two functions.
 
 - `new_path_recognizer` creates a `Recognizer` from a config object containing `patterns` and `generate` fields. See the [example](#example) above for basic usage.
   - Type:
-    ```
+
+```lua
```

Comment:
Please don't un-indent this, this code block belongs to the `Type:` bullet above, but now it looks detached from the bullet due to the lack of indentation.

Also, this is not actually Lua code, it is psuedocode based on the grammar described earlier in the file.

![CleanShot 2024-03-29 at 21 05 34@2x](https://github.com/sourcegraph/docs/assets/93103176/165473e1-b51e-4595-8fee-1b001db66bfa)

Because of this, the indentation for the "where `index_job` is an object with the following shape" below this also becomes incorrect.